src/main/java/com/metamx/emitter/core/HttpEmitterConfig.java

```diff
@@ -61,6 +62,9 @@
   @JsonProperty
   private long maxBufferSize = DEFAULT_MAX_BUFFER_SIZE;
 
+  @JsonProperty
+  private boolean compress = DEFAULT_COMPRESS;
```

Comment:
Such a thing would hopefully be able to fail early  so a bad config won't be propagated until its needed.